src/editors/sharedComponents/TypeaheadDropdown/index.jsx

```diff
@@ -220,6 +220,7 @@ class TypeaheadDropdown extends React.Component {
             data-testid="dropdown-container"
             className="dropdown-container mt-2 rounded bg-light-100 box-shadow-centered-1 mr-2"
             style={{ maxHeight: '300px', overflowY: 'scroll' }}
+            role="listbox"
```

Comment:
nit: i think you probably added this to add test, but seems no longer needed? Unless it is good for a11y?